### PR TITLE
refactor(ui5-label): wrap text by default

### PR DESCRIPTION
**Pull request by dimovpetar**
The text of `ui5-label` now wraps by default.

BREAKING CHANGE: `wrapping-type` property default value has changed from `None` to `Normal`.
Before: 
```html
<ui5-label>some very very very long label</ui5-label> <!-- would truncate the text if there is not enough space -->
```

Now:
```html
<ui5-label>some very very very long label</ui5-title> <!-- would let the text wrap if there is not enough space -->
```

Related to https://github.com/SAP/ui5-webcomponents/issues/8461

### CODE REVIEW

**Comment by alexandar-mitsev** [comment]
LGTM , waiting for UA approval

File: packages/main/test/pages/Kitchen.html [hunk 1]
```diff
@@ -501,8 +501,8 @@
 		</section>
 
 		<section class="row row-centered">
-			<ui5-label>This is a long long even longer text defined via the ui5-label and it truncates by default</ui5-label>
-			<ui5-label wrapping-type="Normal">This is a long long even longer text defined via the ui5-label, but this one wraps</ui5-label>
+			<ui5-label>This is a long long even longer text defined via the ui5-label and it wraps by default</ui5-label>
+			<ui5-label wrapping-type="None">This is a long long even longer text defined via the ui5-label, but it truncates</ui5-label>
```
Comment (BorisDafov) [inline packages/main/test/pages/Kitchen.html:505]: ..., but it's truncated

File: packages/main/test/pages/Kitchen.openui5.html [hunk 2]
```diff
@@ -434,8 +434,8 @@
 		</section>
 
 		<section class="row row-centered">
-			<ui5-label>This is a long long even longer text defined via the ui5-label and it truncates by default</ui5-label>
-			<ui5-label wrapping-type="Normal">This is a long long even longer text defined via the ui5-label, but this one wraps</ui5-label>
+			<ui5-label>This is a long long even longer text defined via the ui5-label and it wraps by default</ui5-label>
+			<ui5-label wrapping-type="None">This is a long long even longer text defined via the ui5-label, but this one truncates</ui5-label>
```
Comment (BorisDafov) [inline packages/main/test/pages/Kitchen.openui5.html:438]: , but this one is truncated

File: packages/website/docs/_components_pages/main/Label.mdx [hunk 3]
```diff
@@ -17,16 +17,16 @@ import CustomStyling from "../../_samples/main/Label/CustomStyling/CustomStyling
 ## More Samples
 
 ### Text Truncation and Wrapping
-The Label truncates by default. To make it wrap - set <b>wrapping-type="Normal"</b>.
+The Label wraps by default. To make it truncate - set <b>wrapping-type="None"</b>.
```
Comment (BorisDafov) [inline packages/website/docs/_components_pages/main/Label.mdx:20]: To truncate it - set ...

File: packages/website/docs/_components_pages/main/Label.mdx [hunk 4]
```diff
@@ -17,16 +17,16 @@ import CustomStyling from "../../_samples/main/Label/CustomStyling/CustomStyling
...
 
 <TextWrapping />
 
 ### Label with Input
-The Labes' <b>for</b>, <b>required</b> and <b>showColon</b> properties can be used to reference inputs.
+The Labels' <b>for</b>, <b>required</b> and <b>showColon</b> properties can be used to reference inputs.
```
Comment (BorisDafov) [inline packages/website/docs/_components_pages/main/Label.mdx:25]: labels (also remove the apostrophe)

File: packages/website/docs/_samples/main/Label/TextWrapping/sample.html [hunk 5]
```diff
@@ -12,9 +12,9 @@
 <body style="background-color: var(--sapBackgroundColor)">
     <!-- playground-fold-end -->
 
-    <ui5-label for="myInputWrapping" class="w200">Label that demonstrates truncation, the long labels till truncate.</ui5-label>
+    <ui5-label class="w200">Label that demonstrates how the long labels could be wrapped. To test the truncation, use wrapping-type="None"</ui5-label>
     <br><br>
-    <ui5-label for="myInputWrapping" wrapping-type="Normal" class="w200">Label that demonstrates how, if set to wrapping-type="Normal", the long labels could be wrapped. To test the truncation, use 'wrapping-type="None"</ui5-label>
+    <ui5-label class="w200" wrapping-type="None">Label that demonstrates truncation, the long labels will truncate.</ui5-label>
```
Comment (BorisDafov) [inline packages/website/docs/_samples/main/Label/TextWrapping/sample.html:17]: will be truncated

File: packages/website/docs/_components_pages/main/Label.mdx [hunk 6]
```diff
@@ -17,16 +17,16 @@ import CustomStyling from "../../_samples/main/Label/CustomStyling/CustomStyling
...
 
 <UsageWithInputs />
 
 ### Custom Styling
-The Labes can be customized with pure CSS, applied on the tag.
+The Labels can be customized with pure CSS, applied on the tag.
```
Comment (BorisDafov) [inline packages/website/docs/_components_pages/main/Label.mdx:30]: labels

File: packages/website/docs/_components_pages/main/Label.mdx (hunk 3 above)
Comment (BorisDafov) [inline packages/website/docs/_components_pages/main/Label.mdx:20]: better:
The ui5-label ...

**Review by BorisDafov — CHANGES_REQUESTED** [review]

File: packages/main/test/pages/Kitchen.openui5.html (hunk 2 above)
Comment (dimovpetar) [reply packages/main/test/pages/Kitchen.openui5.html]: Done

File: packages/main/test/pages/Kitchen.html (hunk 1 above)
Comment (dimovpetar) [reply packages/main/test/pages/Kitchen.html]: Done

**Review by BorisDafov — COMMENTED** [review]
I approve the PR from UA perspective.

**Review by alexandar-mitsev — APPROVED** [review]

**Pull request merged**